rlclientlib/rlclientlib.vcxproj

```diff
@@ -119,7 +119,7 @@ $(flatcPath) -o $(SolutionDir)rlclientlib\generated\v2\ --cpp $(SolutionDir)rlcl
     <PreBuildEvent>
       <Command>$(flatcPath) -o $(SolutionDir)rlclientlib\generated\v1\ --cpp $(SolutionDir)rlclientlib\schema\v1\Metadata.fbs $(SolutionDir)rlclientlib\schema\v1\OutcomeEvent.fbs $(SolutionDir)rlclientlib\schema\v1\RankingEvent.fbs $(SolutionDir)rlclientlib\schema\v1\DecisionRankingEvent.fbs $(SolutionDir)rlclientlib\schema\v1\SlatesEvent.fbs</Command>
       <Command>$(flatcPath) -o $(SolutionDir)rlclientlib\generated\v1\ --cpp $(SolutionDir)rlclientlib\schema\v1\Metadata.fbs $(SolutionDir)rlclientlib\schema\v1\OutcomeEvent.fbs $(SolutionDir)rlclientlib\schema\v1\RankingEvent.fbs $(SolutionDir)rlclientlib\schema\v1\DecisionRankingEvent.fbs $(SolutionDir)rlclientlib\schema\v1\SlatesEvent.fbs
-$(flatcPath) -o $(SolutionDir)rlclientlib\generated\v2\ --cpp $(SolutionDir)rlclientlib\schema\v2\Metadata.fbs $(SolutionDir)rlclientlib\schema\v2\OutcomeEvent.fbs $(SolutionDir)rlclientlib\schema\v2\CbEvent.fbs $(SolutionDir)rlclientlib\schema\v2\CaEvent.fbs $(SolutionDir)rlclientlib\schema\v2\MultiSlotEvent.fbs $(SolutionDir)rlclientlib\schema\v2\Event.fbs $(SolutionDir)rlclientlib\schema\v2\DedupInfo.fbs</Command>
+$(flatcPath) -o $(SolutionDir)rlclientlib\generated\v2\ --cpp $(SolutionDir)rlclientlib\schema\v2\Metadata.fbs $(SolutionDir)rlclientlib\schema\v2\OutcomeEvent.fbs $(SolutionDir)rlclientlib\schema\v2\CbEvent.fbs $(SolutionDir)rlclientlib\schema\v2\CaEvent.fbs $(SolutionDir)rlclientlib\schema\v2\MultiSlotEvent.fbs $(SolutionDir)rlclientlib\schema\v2\Event.fbs $(SolutionDir)rlclientlib\schema\v2\DedupInfo.fbs $(SolutionDir)rlclientlib\schema\v2\LearningModeType.fbs</Command>

```

Comment:
I believe there are 3 other configurations that have to be updated (x32/x64 x Debug/Release)

Reply:
done